apps/api/src/app/trial/trialWithResults/trial-result.service.ts

```diff
@@ -0,0 +1,103 @@
+import {Injectable, OnModuleInit} from "@nestjs/common";
+import {EventService} from "../../event/event.service";
+import {PrismaService} from "../../prisma.service";
+import {ModuleRef} from "@nestjs/core";
+import {trialWithResults, trialWithResultsMiner} from "./trialWithResults";
+import axios from "axios";
+import {event_type, user} from "@prisma/client";
+
+@Injectable()
+export class TrialResultService implements OnModuleInit {
+
+  private eventService: EventService;
+
+  constructor(
+    private readonly prismaService: PrismaService,
+    private readonly moduleRef: ModuleRef
+  ) {
+  }
+
+  onModuleInit(): any {
+    this.eventService = this.moduleRef.get(EventService, {strict: false});
+  }
+
+  async findAllWithResults(): Promise<trialWithResults[]> {
+    const trials = await this.prismaService.trial.findMany();
+    let trialResults: trialWithResults[] = [];
+    let matchResults: any;
+    if (trials.length > 0) {
+      try {
+        const url = `${process.env.REACT_APP_MM_API_URL}` + '/ctims_trial_summary';
```

Comment:
let's not call it REACT_APP_MM_API_URL since technically it is used in Nest.js. Maybe just as simple as MM_API_URL?